tests/cisco/test_cisco_switch_protocol.py

```diff
@@ -1076,6 +1076,27 @@ def test_ip_helper(self, t):
         configuring(t, do="no interface vlan 4000")
 
     @with_protocol
+    def test_ip_route(self, t):
+        enable(t)
+        t.write("configure terminal")
+        t.readln("Enter configuration commands, one per line.  End with CNTL/Z.")
+        t.read("my_switch(config)#")
+        t.write("ip route 1.1.1.0 255.255.255.0 2.2.2.2")
+        t.read("my_switch(config)#")
```

Comment:
Yup, you can replace lines 1081 to 1085 by configuring(t, do="ip route 1.1.1.0 255.255.255.0 2.2.2.2")